workflow-runtime/src/jvmTest/kotlin/com/squareup/workflow1/StressTestHelpers.kt

```diff
@@ -0,0 +1,27 @@
+package com.squareup.workflow1
+
+import java.util.concurrent.CountDownLatch
+
+/**
+ * Returns the maximum number of threads that can be ran in parallel on the host system, rounded
```

Comment:
```suggestion
 * Returns the maximum number of threads that can be run in parallel on the host system, rounded
```

:D

Reply:
After August 1 there's gonna be so many more "ran" in the codebases :P

Reply:
Bonus points if one of the last things you do is write a lint check that catches this.

Reply:
Oh my God.

Oh no.

Reply:
Done.